css-view-transitions-1/Overview.bs

```diff
@@ -1414,12 +1424,6 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 				1. Append |new| to |imagePair|.
 
-				The [=new element=] and its contents
```

Comment:
Why remove this? The earlier note is non-normative IIUC so this can be the technical definition.

The text above also misses the point about hit-testing but its fine if that's covered here.

Reply:
Discussed offline.